ast/ast-common/src/main/kotlin/kastree/ast/MutableVisitor.kt

```diff
@@ -270,6 +270,12 @@ open class MutableVisitor {
                         anns = visitChildren(anns, newCh),
                         expr = visitChildren(expr, newCh)
                     )
+                    is Node.Expr.TypeOp -> copy (
+                        lhs = visitChildren(lhs, newCh),
+                        oper = visitChildren(oper, newCh),
+                        rhs = visitChildren(rhs, newCh)
+                    )
+                    is Node.Expr.TypeOp.Oper -> this
```

Comment:
I tried a couple visitChildren variants here, but couldn't get it to work, well.  

I would assume that converting `as` to `as?` could be a valid case and this doesn't necessarily block that.